jobs/kpi-forecasting/kpi-forecasting/yaml/desktop.yaml

```
@@ -11,8 +11,8 @@ columns:
 - submission_date
 - qcdou
 write_project: "moz-fx-data-bq-data-science"
```

Should this be `moz-fx-data-shared-prod`?

```suggestion
write_project: "moz-fx-data-shared-prod"
```